forest/src/cli/chain_cmd.rs

```diff
@@ -53,7 +53,7 @@ pub enum ChainCommands {
         cid: String,
     },
 
-    /// Reads and prints out ipld nodes referenced by the specified CID from chain
+    /// Reads and prints out `ipld` nodes referenced by the specified CID from chain
```

Comment:
```suggestion
    /// Reads and prints out `IPLD` nodes referenced by the specified CID from chain
```

Reply:
IPLD acronyms is already in the dictionary.

Reply:
:+1: